config.tf

```diff
@@ -1,6 +1,14 @@
+locals {
+  app_data_mode   = "${var.postgresql_address != "" ? "external_services" : "demo"}"
+  
+  or something to say what it is better.
+  app_network_type     = "${var.airgap_package_url != "" ? "airgap" : "online" }"
+  install_type = "${local.demo_or_es}-${local.ag_or_ol}"
```

Comment:
this also needs to be changed if you update the previous two

Reply:
Yup, done.